### PR TITLE
[terra-clinical-item-display] Update text style meaning to use VisuallyHiddenText

### PR DESCRIPTION
**Pull request by smason0**
### Summary

**What was changed:**
VisuallyHiddenText now "wraps" the ItemDisplay text to provide screen reader support for text styling. Removes the previous implementation of using an aria-label on a parent span element. See original change here: https://github.com/cerner/terra-clinical/pull/843

**Why it was changed:**
Use of aria-label is not recommended for non-interactive elements, such as divs and spans, or elements without roles. 

### Testing

This change was tested using:

- [x] WDIO
- [x] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [x] Other (please describe below)
- [ ] No tests are needed

Tested with JAWS in Edge browser and with VoiceOver in Safari, Chrome, and Firefox.

### Reviews

In addition to engineering reviews, this PR needs:

- [x] UX review
- [x] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9060 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by sycombs — APPROVED** [review]

File: packages/terra-clinical-item-display/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
   * `strong` tag around strong style text.
   * `span` tag around non-strong style text.
 
+* Changed
+  * Make screen reader support for text styling more robust by using terra-visually-hidden-text.
```
Comment (sdadn) [inline packages/terra-clinical-item-display/CHANGELOG.md:10]: Nitpicky but how does this sound?
```suggestion
  * Improved screen reader support for text styling by utilizing terra-visually-hidden-text.
```

Comment (smason0) [reply packages/terra-clinical-item-display/CHANGELOG.md]: Done: https://github.com/cerner/terra-clinical/pull/855/commits/106c28265abb46338f0d525e4e02e91657637d29

**Review by artpark — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

File: packages/terra-clinical-item-display/src/ItemDisplay.jsx [hunk 2]
```diff
@@ -115,27 +116,24 @@ const ItemDisplay = ({
...
-          <div data-terra-clinical-item-display-text className={textClassNames} aria-hidden="true">
-            {textWrapper}
-          </div>
-        </span>
-      ) : (
-        <div data-terra-clinical-item-display-text className={textClassNames}>
-          {textWrapper}
-        </div>
-      )}
+      {hiddenStyleMeaning ? <VisuallyHiddenText text={hiddenStyleMeaning} /> : null}
```
Comment (trandrew1023) [inline packages/terra-clinical-item-display/src/ItemDisplay.jsx:132]: nit: I believe this works the same? Also maybe the same on line 136

```suggestion
      {hiddenStyleMeaning && <VisuallyHiddenText text={hiddenStyleMeaning} />}
```

**Review by trandrew1023 — APPROVED** [review]

**Review by eawww — APPROVED** [review]

**Comment by mjpalazzo** [comment]
Nice work, Steven. I validated the Comment Item example with disabled text with JAWS and VoiceOver. Both ATs announced "Comment."

**Pull request merged**